utils/fileCrud.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"context"
+	"io"
+	"mime/multipart"
+	"time"
+)
+
+const (
+	bucketName = "pmc-server-img"
+)
+
+func UploadFile(file multipart.File, uploadPath string, fileName string) (string, error) {
+	ctx := context.Background()
+
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	wc := storageClient.cl.Bucket(bucketName).Object(uploadPath + fileName).NewWriter(ctx)
```

Comment:
```suggestion
	wc := storageClient.cl.Bucket(bucketName).Object(uploadPath + "/" + fileName).NewWriter(ctx)
```
This might help fix uploading outside of team_pictures folder.